cli/command/service/opts.go

```diff
@@ -165,7 +165,7 @@ func updateConfigFromDefaults(defaultUpdateConfig *api.UpdateConfig) *swarm.Upda
 }
 
 func (opts updateOptions) updateConfig(flags *pflag.FlagSet) *swarm.UpdateConfig {
-	if !anyChanged(flags, flagUpdateParallelism, flagUpdateDelay, flagUpdateMonitor, flagUpdateFailureAction, flagUpdateMaxFailureRatio) {
+	if !anyChanged(flags, flagUpdateParallelism, flagUpdateDelay, flagUpdateMonitor, flagUpdateFailureAction, flagUpdateMaxFailureRatio, flagUpdateOrder) {
```

Comment:
Could you add a small unit test before this one, to test the changes? https://github.com/docker/cli/blob/88c6089300a82d3373892adf6845a4fed1a4ba8d/cli/command/service/opts_test.go#L230

Something like;

```go

func TestToServiceUpdateRollbackOrder(t *testing.T) {
	flags := newCreateCommand(nil).Flags()
	flags.Set("update-order", "start-first")
	flags.Set("rollback-order", "stop-first")

	o := newServiceOptions()
	o.mode = "replicated"
	o.update = updateOptions{order: "start-first"}
	o.rollback = updateOptions{order: "stop-first"}

	service, err := o.ToService(context.Background(), &fakeClient{}, flags)
	assert.NilError(t, err)
	assert.Check(t, is.Equal(service.UpdateConfig.Order, o.update.order))
	assert.Check(t, is.Equal(service.RollbackConfig.Order, o.rollback.order))
}
```

Reply:
I see! I will a test for it.
Thanks for your comment! 👍

Reply:
I have added a test function.
Please take a look at it!